src/Components/README.md

```diff
@@ -35,6 +35,36 @@ To build this specific project from source, follow the instructions [on building
 
 **Note:** You also need to run the preceding `build` command in the command line before building in VS to ensure that the Web.JS dependency is built.
 
+### A Guide to build Web.JS
+
+Prior to building `src\Components\Web.JS\`
+
+1. You'll need to install [Node](https://nodejs.org/en/) and [yarn](https://yarnpkg.com/) on your machine.
+2. You'll need to run the `restore` script locally to install the required dotnet dependencies and setup the repo. The `restore` script is located in the root of the repo.
+
+```bash
+./restore.sh
+```
+
+```powershell
+./restore.ps1
+```
+
+3. After the restore script has finished executing, activate the locally installed .NET by running the following command.
+
+```bash
+source activate.sh
+```
+
+```powershell
+. ./activate.ps1
+```
+
+Now you can build `src\Components\Web.JS\` by running the following command in the directory.
+```powershell
+dotnet build
```

Comment:
Is dotnet build sufficient in all cases? I thought we had to do both the dotnet build and yarn build to ensure all the bits (including the JS Interop bits) get updated.

```suggestion
dotnet build
yarn build
```

Reply:
After running the activate script, `dotnet build` will run `yarn build` like this

```powershell
dotnet build
MSBuild version 17.4.0-preview-22479-02+88f89b105 for .NET
  Running yarn install on {path}\src\Components\Web.JS\Microsoft.AspNetCore.Compon
  ents.Web.JS.npmproj
  yarn install v1.22.10
  [1/4] Resolving packages...
  success Already up-to-date.
  Done in 0.18s.
  yarn run v1.22.10
  $ yarn run clean && yarn install --mutex network --frozen-lockfile
  $ yarn install --mutex network --frozen-lockfile
  [1/4] Resolving packages...
  success Already up-to-date.
  $ node node_modules/rimraf/bin.js ./dist/Debug ./dist/Release
  [1/4] Resolving packages...
  success Already up-to-date.
  $ yarn run build:debug && yarn run build:production
  $ cd src && node ../node_modules/webpack-cli/bin/cli.js --mode development --config ./webpack.config.js
```

Without activating `dotnet build` will not run:
```powershell
The command could not be loaded, possibly because:
  * You intended to execute a .NET application:
      The application 'build' does not exist.
  * You intended to execute a .NET SDK command:
      A compatible .NET SDK was not found.

Requested SDK version: 8.0.100-alpha.1.22480.9
global.json file: {path}\aspnetcore\global.json

Installed SDKs:
6.0.402 [C:\Program Files\dotnet\sdk]

Install the [8.0.100-alpha.1.22480.9] .NET SDK or update [{path}\aspnetcore\global.json] to match an installed SDK.

Learn about SDK resolution:
https://aka.ms/dotnet/sdk-not-found
```